app/config/security.yml

```diff
@@ -111,6 +111,7 @@ security:
             - ROLE_APP_ADMIN_CITIZEN_PROJECT_VIEW
             - ROLE_APP_ADMIN_CITIZEN_PROJECT_EDIT
             - ROLE_APP_ADMIN_CITIZEN_PROJECT_DELETE
+            - ROLE_APP_ADMIN_CITIZEN_PROJECT_CATEGORY_ALL
```

Comment:
Why do we define a new role if it is not used ?

Reply:
Sonata will dynamically use this role for the CitizenProjectCategoryAdmin